scripts/reboot

```diff
@@ -78,6 +79,17 @@ function stop_sonic_services()
     stop_pmon_service
 }
 
+function stop_services_asan()
+{
+    if [[ x"$ASIC_TYPE" == x"mellanox" ]]; then
+        debug "Stopping syncd for ASAN"
+        systemctl stop syncd
```

Comment:
Stopping SWSS should also stop syncd automatically, is it necessary to explicitly stop syncd here?

Reply:
Good catch, thanks. I've updated the PR: https://github.com/Azure/sonic-utilities/pull/2107/commits/5484b43d83a50de6217c63cef9290fe9830c2068